libcxx/utils/sym_diff.py

```diff
@@ -80,6 +80,11 @@ def main():
         old_syms_list, _ = util.filter_stdlib_symbols(old_syms_list)
         new_syms_list, _ = util.filter_stdlib_symbols(new_syms_list)
 
+    for symbol in new_syms_list:
+        if 'B' in symbol["name"]:
+            print(f"Symbol {symbol["name"]} contains an ABI tag!")
+            sys.exit(1)
```

Comment:
Would it be possible to add this as a `lit` test instead? One thing we could potentially do is `nm -g %{lib-dir}/libc++.dylib | grep -v 'B'` or something along those lines?

Reply:
Since the name of the library changes depending on the platform this is non-trivial I think. Do you have any idea how to get that except passing it through as another lit substitution?

Reply:
That's right, this is kinda tricky. I had plans to migrate the ABI list tests to Lit at some point, but this is one of the difficulties I encountered too. I think this is good enough for now. For anyone curious: https://github.com/ldionne/llvm-project/commit/060c7d6e5b27ced5757c4fd3da0754d6356ea2b4.